common/scripts/build-all.sh

```diff
@@ -41,22 +41,29 @@ PACKAGE=$2
 
 source ./build-scripts/build-uefi.sh
 
-if [ $BAND == "SR" ]; then
-    source ./build-scripts/build-sbsaefi.sh
+if [[ $ARCH = "arm" ]]; then
```

Comment:
The changes for $ARCH = "arm", may be moved to a new file
build-all-arm.sh

This file may be called from build-ir-live-image.sh if $ARCH="arm"